### PR TITLE
Try fixing the sporadic derived params aliases failure

### PR DESCRIPTION
**Pull request by taldcroft**
## Description

This attempts to fix the  sporadic derived params aliases failure, and provides better debug info if it still fails.

## Interface impacts

None.

### Unit tests

- [x] Mac

### Functional tests

No functional testing.

### CODE REVIEW

**Comment by jeanconn** [comment]
Have you been able to get insight into why it has been failing?  Also, what is that test really testing?

**Comment by taldcroft** [comment]
> Have you been able to get insight into why it has been failing? 

No, hence the "just try something different" approach.

**Comment by taldcroft** [comment]
The test is ensuring that some MSID munging is working for CXC and MAUDE. It is not strictly necessary to check that some data was fetched, but it seems like a good thing.

**Comment by jeanconn** [comment]
My confusion was about the test trying to get overlapping time ranges from maude and eng archive and what that was testing (explicitly or implicitly).  Otherwise wondering if it could test more separately.

**Comment by taldcroft** [comment]
Yes, it's all a bit confusing. This is a tiny unimportant test, so let's not spend too much more time on this discussion and just see if this works.

File: Ska/engarchive/tests/test_fetch.py [hunk 1]
```diff
@@ -156,15 +157,20 @@ def test_filter_bad_times_default_copy():
...
     assert d2.MSID == d1.MSID  # normalized version for accessing databases
     assert np.all(d1.times == d2.times)
     assert np.all(d1.vals == d2.vals)
-    assert len(d1) > 0
+    if len(d1) == 0:
+        msg = (f'No data found for cxo_tstop={CxoTime(cxc_tstop).date} {dt=}'
+               f' {sources=} {msid=} '
+               f'{CxoTime(cxc_tstop - dt).date=} '
+               f'{CxoTime(cxc_tstop + dt).date=}')
+        raise AssertionError(msg)
```
Comment (javierggt) [inline Ska/engarchive/tests/test_fetch.py:173]: When I was looking into this error, I did something similar to this, and I ended progressively increasing the window. Maybe we can save the time and look for the last time with data, and report that.

**Review by jeanconn — APPROVED** [review]

**Pull request merged**